src/server/task-list/controller.test.js

```diff
@@ -72,6 +72,17 @@ describe('#taskListController', () => {
   })
 
   it('Should return the correct task list items when there is state enough to show all sections', async () => {
+    const { origin, destination } = applicationStateWithAnimalIdentifiersSection
+    await session.setState('origin', {
```

Comment:
I think these overrides can be removed now?  because they match what is currently in `applicationStateWithAnimalIdentifiersSection`

Reply:
done